functions.R

```diff
@@ -0,0 +1,99 @@
+library(tidyverse)
+library(janitor)
+
+data_dictionary <- list.files("data", "FirstResponderCOVID19TestingPr")
+data_dictionary <- read.csv(paste0("data/", data_dictionary)) %>% 
+  clean_names()
```

Comment:
This file is not included in your PR. @pbchase suggests fetching this with the API if possible.

Reply:
I will look into reading the data dictionary with the API later. For now we can download it manually as this needs to go out soon.